elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -1,200 +1,359 @@
-    background:   var(--pfe-cta--aux);
-    color:        var(--pfe-cta--main);
+::slotted(:hover) {
+    color: pfe-local(Color--hover) !important; // @WHY redhat-theme styles on links
+}
 
-    --pfe-broadcasted--color--ui-link--hover: var(--pfe-cta--main);
+::slotted(:focus) {
+  outline: none !important;
+  color: pfe-local(Color--focus) !important;  // @WHY redhat-theme styles on links
 
-    &::after {
-      display: none;
-    }
+  &::before {
+    border: 1px solid #{pfe-local($cssvar: BorderColor--focus, $region: inner)};
   }
 
-  ::slotted(a:hover), // Not sure we can lump these together
-  ::slotted(a:focus) {
-    border-color: var(--pfe-cta--main--hover);
-    background:   var(--pfe-cta--main--hover);
-    color:        var(--pfe-cta--aux--hover);
-    
-    --pfe-broadcasted--color--ui-link--hover: var(--pfe-cta--aux--hover);
+  .pfe-cta--arrow {
+    fill: pfe-local(Color--focus);
   }
+}
+
+
 
-  // & ::slotted(a:visited),   TBD
+/// ===========================================================================
+/// BELOW THIS POINT, LOCAL VARIABLE OVERRIDES ONLY
+
+
+// Broadcast colors: The CTA is tuning into whatever colors the containers are offering up, per priority level 
+
+// - The location of these styles in the doc is important for clarity (readable code FTW!)
+// - pfe-priority="primary" sets its own background colors, so it ignores broadcast 
+// - if there is no broadcast variable available, these are still seen by the browser, so we also set fallbacks here!
+// - Fallback values should match whats above
+
+// default link background focus color
+:host(:not([pfe-priority])) {
+  --pfe-cta--Color: var(--pfe-broadcasted--color--ui-link, #{pfe-color(ui-link)});
+
+  --pfe-cta--inner--BorderColor--hover: var(--pfe-broadcasted--color--ui-link--hover, #{pfe-color(ui-link--hover)});
+  --pfe-cta--Color--hover: var(--pfe-broadcasted--color--ui-link--hover, #{pfe-color(ui-link--hover)});
+
+  --pfe-cta--BackgroundColor--focus: #{$pfe-cta--BackgroundColor--focus}; // @NOTE Definition at top of file
+  --pfe-cta--Color--focus: var(--pfe-broadcasted--color--ui-link--focus, #{pfe-color(ui-link--focus)});
+}
+  
+:host([pfe-priority="secondary"]) {
+  --pfe-cta--BorderRadius:              #{pfe-var(ui--border-radius)};
+  --pfe-cta__arrow--Display:            none;
+  --pfe-cta--BackgroundColor:           transparent;
+  --pfe-cta--BorderColor:               var(--pfe-broadcasted--color--text, #{pfe-color(ui-complement)});
+  --pfe-cta__inner--BorderColor:        transparent;
+  --pfe-cta--Color:                     var(--pfe-broadcasted--color--text, #{pfe-color(ui-complement)});
+
+  --pfe-cta--BackgroundColor--hover:    #{pfe-color(ui-complement)};
+  --pfe-cta--BorderColor--hover:        var(--pfe-broadcasted--color--text, #{pfe-color(ui-complement)}); //white, then black
+  --pfe-cta--Color--hover:              #{pfe-color(ui-complement--text)};
+
+  --pfe-cta--BackgroundColor--focus:    #{pfe-color(surface--lighter)};
```

Comment:
@starryeyez024 this is the only color (it's a pale gray, #f0f0f0) for which we don't have anything close in the `ui-complement` family. Should I add a `ui-complement--focus` variable for this?

Reply:
I'm feeling like we could merge this as-is since it works, and then go back and update color variables here?

Reply:
Okay, works for me.